source/assets/critical/c_landingpage.scss

```diff
@@ -13,8 +13,11 @@ body.body--loading.body--loading {
   .h-loading-only {
     display: block;
   }
-
   .collection {
+    margin: 0 auto;
+    max-width: $desktop-wide-size;
+  }
```

Comment:
noch ein Abständli danach und gut ist

Reply:
da gibt's in diesem file noch weitere solche stellen - ich bin bereits in einem anderen branch daran, das critical css umzubauen.